backend/app.js

```diff
@@ -23,15 +24,11 @@ app.use(addCrossDomainHeaders)
 app.use(urlencodedParser)
 app.use(jsonParser)
 
-let bot 
-botFactory.makeBot(process.env.FOOSBOT_TOKEN, process.env.FOOS_CHANNEL_NAME)
-    .then(resolvedBot => {
-        bot = resolvedBot
-        console.log('Slackbot initialized!')
-    })
-    .catch((error) => console.log('Bot initialization failed', error))
-
-
+const matchReporter = new MatchReporter(
+  process.env.FOOSBOT_TOKEN,
```

Comment:
i'd like to think of a bot as a separate entity. potentially used for recording a match, responding to various stats upon request etc. - it shuold be initialized within the match reporter but separtely